website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -108,23 +108,334 @@ $ curl \
+| `organization_task_url`      | string             | The URL associated with the organization task.           |
+
+
+#### Create
+
+A Terraform Cloud organization emits this event when someone in your organization creates a run task for a workspace (`workspace_task`). Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key                      | Data Type    | Description                     |
+| ---                      | ---          | ---                             |
+| `task_enforcement_level` | string       | The task's enforcement level.    |
+| `task_stage`             | string       | The stage of the task.           |
+| `workspace`              | string       | The workspace associated with the task. |
+| `workspace_id`           | string       | The ID of the workspace.         |
+| `organization_task`      | string       | The organization's task.         |
+| `organization_task_id`   | string       | The ID of the organization's task.|
+
+#### Update
+
+A Terraform Cloud organization emits this event when someone in your organization updates a run task for a workspace (`workspace_task`). Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key                      | Data Type    | Description                     |
+| ---                      | ---          | ---                             |
+| `task_enforcement_level` | string       | The task's enforcement level.    |
+| `task_stage`             | string       | The stage of the task.           |
+| `workspace`              | string       | The workspace associated with the task. |
+| `workspace_id`           | string       | The ID of the workspace.         |
+| `organization_task`      | string       | The organization's task.         |
+| `organization_task_id`   | string       | The ID of the organization's task.|
+
+#### Destroy
+
+A Terraform Cloud organization emits this event when someone in your organization destroys a run task for a workspace (`workspace_task`). Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key                      | Data Type    | Description                     |
+| ---                      | ---          | ---                             |
+| `task_enforcement_level` | string       | The task's enforcement level.    |
+| `task_stage`             | string       | The stage of the task.           |
+| `workspace`              | string       | The workspace associated with the task. |
+| `workspace_id`           | string       | The ID of the workspace.         |
+| `organization_task`      | string       | The organization's task.         |
+| `organization_task_id`   | string       | The ID of the organization's task.|
+
+### Task events
+
+TODO: Description of what a task is.
```

Comment:
@carolinaborim I wasn't sure what a task actually was in this scenario, they didn't seem like run tasks? So I wasn't sure. Please feel free to suggest a description or commit directly on this branch!

Reply:
Yes, the task refers to a run task.

Reply:
How does a workspace's run task (`workspace_task`) differ from the normal `task`? They both have CUD, so I was a bit confused how to represent them. Is `workspace_task` a task associated with a specific workspace?

Reply:
I mentioned on the spreadsheet, but will copy and paste here to have all the info in only one place :)

> Thanks for the question. A task is a Run Task created/edited/deleted and it is on organisation level. When we refer to Workspace Task, we are talking about a Run Task that was associated to some Workspace. ([https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks#associating-run-tasks-with-a-workspace](https://www.google.com/url?q=https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks%23associating-run-tasks-with-a-workspace&sa=D&source=editors&ust=1706519791709869&usg=AOvVaw1-OnU5TyyhrVbUWGtDb28f))

Reply:
[Using the API definition](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks), we could define something like this:
Run tasks are tasks created on organisation level that let you directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. You can use run tasks to validate Terraform configuration files, analyze execution plans before applying them, scan for security vulnerabilities, or perform other custom actions.

Reply:
Sounds good! Added a few slight rephrases for active voice:
You can create Terraform Cloud [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) at the organization-level to directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. Use run tasks to validate Terraform configuration files, analyze execution plans before applying them, scan for security vulnerabilities, or perform other custom actions.

Reply:
I also added a link to the `Create a workspace's run task` header with the link to the docs you provided above! Thank you for the context here ✨